Add more context on Conflict/serialisation error (#544)
memgraph/documentation: pages/help-center/errors/conflicting-transactions.mdx

---
title: Conflicting transactions
description: Learn about conflicting transaction and serialization errors as well as troubleshooting best practices for serialization errors on our all-in-one documentation page at Memgraph. 
---

import { Callout } from 'nextra/components'

# Conflicting transactions

<Callout type="info">
If you are encountering issues with conflicting transactions or serialization errors, we're here to help. Join the conversation on <a href="https://www.discord.gg/memgraph" target="_blank">Discord</a> for support.
</Callout>

## Error messages

1. **Unable to commit due to serialization error.**
2. **Cannot resolve conflicting transactions.**

## Conflicting transaction and serialization error

> Conflicting transactions and serializations are, in a sense, the same error that usually occurs when concurrent data 

Reviewer: ```suggestion
Conflicting transactions and serialization errors are, in a sense, the same error that usually occurs when concurrent data 
```

imports and transactions try to work on the same data points simultaneously. This situation can occur in both testing 
and production environments. It's important that your code anticipates and handles these errors. 

A common scenario that triggers an error is when two nodes in a graph are being updated by two
different transactions attempting to add relationships between them at the same moment, changing a property, label, etc. 
The same applies to relationships as well. In essence, it is a write-write conflict.

> For example, if both transactions try to add an edge at the same time to 

Reviewer: ```suggestion
For example, if both transactions try to add a relationship at the same time to 
```
